tests/python/test_struct_for_intermediate.py

```diff
@@ -12,7 +12,7 @@ def _test_nested():
 
     @ti.kernel
     def iterate():
-        for i, j in x.parent():
+        for i, j in x._parent():
```

Comment:
This is a huge break of user code. Let's go back to `parent()`.

Reply:
done